src/views/SystemProfile/index.tsx

```diff
@@ -178,8 +178,8 @@ const SystemProfile = () => {
               </BreadcrumbBar>
 
               <PageHeading className="margin-top-2">
-                <BookmarkCardIcon
-                  size="sm"
+                <IconBookmark
```

Comment:
This icon renders as black and smaller than it should be.  I think the component should be rendered like this -

`<IconBookmark size={4} className="text-primary" />`